Bottom: 563d070aeac350e2fd7e281a380c6c137820d339
Top:    3ddf1c2d138d3077701ee81278156332bbd268fc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 14:37:56 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 33da5f6..1cd0c28 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1595,6 +1595,12 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
    *
    */
 
+#if 1
+    fprintf (stderr, "Subtracting pin + via drills\n");
+    r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+    r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+#endif
+
   if (info.poly == NULL)
     {
       fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
